src/Libraries/RevitNodes/GeometryConversion/UnitConverter.cs

```diff
@@ -24,6 +24,41 @@ public static double DynamoToHostFactor(ForgeTypeId specTypeId)
             return UnitUtils.ConvertToInternalUnits(1, unitTypeId);
         }
 
+        public static ForgeTypeId ConvertRevitInternalUnitTypeIdToSupportedUnitTypeId(ForgeTypeId forgeTypeId)
```

Comment:
Is this an issue for Revit API? For the following types, is "UnitTypeId.FeetFractionalInches" and "UnitTypeId.StationingFeet" the same as "UnitTypeId.Feet" in Revit?

Another question, is this should cherry pick to Revit 2023?

Reply:
I think it is not an issue when using the Revit API by itself.  It only becomes an issue when converting between Revit API units and those found in the ForgeUnitSDK schemas.  There seem to be a few special cases but  I am going to confirm with the Omicron team.  

I would say yes to cherry pick once this PR is approved